### PR TITLE
Much bigger gridsearch

### PR DESCRIPTION
**Pull request by turian**
We won't use this for the final leaderboard. Instead, we'll make a much tighter grid by running these on strong benchmarks.

### CODE REVIEW

File: heareval/predictions/task_predictions.py [hunk 1]
```diff
@@ -742,13 +752,26 @@ def print_scores(mode, scores_and_trainers):
         scores_and_trainers.append((best_model_score, trainer, predictor))
         print_scores(mode, scores_and_trainers)
 
+    def vstr(v):
```
Comment (jorshi) [inline heareval/predictions/task_predictions.py:755]: vstr stands for value str? This is for serializing values to prepare for json? I know you want this short but the intentions of this function aren't super clear. `serialize_value(str: v)`?

File: heareval/embeddings/task_embeddings.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 from torch.utils.data import DataLoader, Dataset
 from tqdm.auto import tqdm
 
-import wandb
+# import wandb
```
Comment (jorshi) [inline heareval/embeddings/task_embeddings.py:41]: What's up with the wandb stuff here?

**Review by jorshi — APPROVED** [review]

File: heareval/predictions/task_predictions.py (hunk 1 above)
Comment (turian) [reply heareval/predictions/task_predictions.py:755]: Done, good idea

File: heareval/embeddings/task_embeddings.py (hunk 2 above)
Comment (turian) [reply heareval/embeddings/task_embeddings.py:41]: I'm sick of auth'ing wand for every GCP run. But when you use it, you get stats charts like the following

image

**Pull request merged**